ghost/src/ohbot.h

```diff
@@ -202,6 +202,7 @@ class COHBot
     uint32_t m_MaxGames;					// config value: maximum number of games in progress
     char m_CommandTrigger;					// config value: the command trigger inside games
     string m_ColoredNamePath;
+    bool m_ColoredNameHide;

```

Comment:
Variable name does not fit to its functionality.
Rename **m_ColoredNameHide;** to **m_EnableColoredNames** in all files and i will merge your request.
Names never get hidden.. they get colored or they dont.